### PR TITLE
Update fp-006-textual-definitions.md

### PR DESCRIPTION
**Pull request by nataled**

### CODE REVIEW

**Review by nataled — COMMENTED** [review]
The last part of the paragraph on agreement between textual and logical definitions needs revision. Perhaps should be revised to "...class is fuzzy or it is not possible to compose suitable equivalence axioms."

File: principles/fp-006-textual-definitions.md [hunk 1]
```diff
@@ -19,14 +19,17 @@ The ontology has textual definitions for the majority of its classes and for top
...
 
 Recommendations and Requirements
 --------------
 
 Textual definitions MUST be unique within an ontology (i.e. no two terms should share a definition). Textual definitions SHOULD follow Aristotelian form (e.g. “a B that Cs” where B is the parent and C is the differentia), where this is practical.
 
-For terms lacking textual definitions, there should be evidence of implementation of a strategy to provide definitions for all remaining undefined terms. In lieu of textual definitions, there can be elucidations when the term can not be rigorously defined.
+For terms lacking textual definitions, there should be evidence of implementation of a strategy to provide definitions for all remaining undefined terms. In lieu of textual definitions, there can be elucidations when the term can not be rigorously defined. Note that textual definitions can be programmatically generated from logical definitions, if available (see [http://oro.open.ac.uk/21501/1/](http://oro.open.ac.uk/21501/1/)).
```
Comment (nicolevasilevsky) [inline principles/fp-006-textual-definitions.md:29]: should we also mention that Dead Simple Ontology Design Patterns (DOSDPs) can be used to generate text and logical definitions:
https://github.com/INCATools/dead_simple_owl_design_patterns
?

**Review by nicolevasilevsky — APPROVED** [review]

File: principles/fp-006-textual-definitions.md (hunk 1 above)
Comment (nataled) [reply principles/fp-006-textual-definitions.md:29]: Good idea! Do you have suitable text?

Comment (nicolevasilevsky) [reply principles/fp-006-textual-definitions.md:29]: How about:

[Dead Simple Ontology Design Patterns (DOSDPs)](https://github.com/INCATools/dead_simple_owl_design_patterns) can be used to generate text and logical definitions. DOSDPs are design specifications, written in yaml format, that specify structured text definitions and logical definitions for groups of ontology terms. These are widely used in many OBO Foundry ontologies, such as [Mondo](https://github.com/monarch-initiative/mondo) and [uPheno](https://github.com/obophenotype/upheno). For some example patterns, see [Mondo patterns](https://mondo.readthedocs.io/en/latest/editors-guide/patterns/) and [uPheno patterns](https://github.com/obophenotype/upheno/tree/master/src/patterns/dosdp-patterns).

**Comment by nataled** [comment]
That works. YAML should probably be in caps?

**Pull request merged**